compiler/luci/import/src/ImporterEx.cpp

```diff
@@ -33,6 +33,17 @@ inline constexpr uint64_t FLATBUFFERS_SIZE_MAX = 2147483648UL; // 2GB
 
 } // namespace
 
+ImporterEx::ImporterEx()
+  : _error_handler{[](const std::exception &e) { std::cerr << e.what() << std::endl; }}
+{
+  // the default error handler above preserves the behavior in ONE and logs all import errors to std::cerr
+}
+
+ImporterEx::ImporterEx(const std::function<void(const std::exception &)> &error_handler)
+  : _error_handler{error_handler}
```

Comment:
can `error_handler` be nullptr? what happens if so?

Reply:
The `error_handler` itself cannot be null since it's an object. I will try to pass a null pointer to a function though and prevent it in the constructor if that's possible.